packages/react/src/themes/teams/components/Dialog/dialogStyles.ts

```diff
@@ -32,6 +32,7 @@ export default {
     gridColumn: '1 / span 2',
     gridRow: 2,
     justifySelf: 'left',
+    width: '100%',
```

Comment:
Did we have some screener diff after this changes? If not, let's add some tests for catching regressions for this in future.